### ISSUE TITLE
Aim assistance's max range is 10

### ISSUE THREAD

**Issue by N3ROO** [N3ROO/Bettercolors#54]
```java
AxisAlignedBB area = new AxisAlignedBB(
        playerX - 10,
        playerY - 10,
        playerZ - 10,
        playerX + 10,
        playerY + 10,
        playerZ + 10
);
```

needs to be changed to

```java
AxisAlignedBB area = new AxisAlignedBB(
        playerX - range,
        playerY - range,
        playerZ - range,
        playerX + range,
        playerY + range,
        playerZ + range
);
```

in `AimAssistance.java`

**Comment by N3ROO** [comment]
Fixed in Bettercolors 6.2.0 by this commit a024f8480b99476af5b93d00b6fa226a03b483b8

**Comment by N3ROO** [comment]
fixed in 7.0.0
